spec/lib/myfinance/resources/receivable_account_spec.rb

```diff
@@ -161,4 +230,154 @@
+        ])
+      end
+    end
+
+    context "when any data is invalid" do
+      let(:params) { { due_date: '2015-10-20', amount: 150.99, create_as_recurrent: "october" } }
+
+      it "raises Myfinance::RequestError" do
+        expect { subject }.to raise_error(Myfinance::RequestError)
+      end
+
+      it "adds information on request error object" do
+        expect(Myfinance::RequestError).to receive(:new).with(code: 422, message: "Unprocessable Entity", body: { "create_as_recurrent"=>["não está incluso na lista"] }).and_call_original
+        expect { subject }.to raise_error(Myfinance::RequestError)
+      end
+    end
+
+    context "when entity does not exist" do
+      subject { client.receivable_accounts.create_as_recurrent(555, {}) }
+
+      it "raises Myfinance::RequestError" do
+        expect { subject }.to raise_error(Myfinance::RequestError)
+      end
+
+      it "adds information on request error object" do
+        expect(Myfinance::RequestError).to receive(:new).with(code: 403, message: "Forbidden", body: {"error" => "Você não tem permissão para acessar este recurso." }).and_call_original
+        expect { subject }.to raise_error(Myfinance::RequestError)
+      end
+    end
+  end
+
+  describe "#create_as_parcelled", vcr: true do
+    let(:params) { { due_date: '2015-10-20', amount: 150.99, create_as_recurrent: "monthly", number_of_parcels: 7 } }
+    subject { client.receivable_accounts.create_as_parcelled(entity_id, params) }
+
+    context "with success" do
+      it "creates a new object" do
+        expect(subject.id).to eq(1298985)
+        expect(subject.due_date).to eq(Date.new(2015, 10, 20))
+        expect(subject.entity_id).to eq(entity_id)
+        expect(subject.status).to eq(1)
+        expect(subject.status_name).to eq('unreceived')
+        expect(subject.occurred_at).to be_nil
+        expect(subject.amount).to eq(150.99)
+        expect(subject.ticket_amount).to be_nil
+        expect(subject.interest_amount).to be_nil
+        expect(subject.discount_amount).to be_nil
+        expect(subject.total_amount).to be_nil
+        expect(subject.description).to be_nil
+        expect(subject.document).to be_nil
+        expect(subject.document_emission_date).to be_nil
+        expect(subject.observation).to be_nil
+        expect(subject.remind).to be_falsy
+        expect(subject.reminded_at).to be_nil
+        expect(subject.income_tax_relevant).to be_falsy
+        expect(subject.category_id).to be_nil
+        expect(subject.classification_center_id).to be_nil
+        expect(subject.expected_deposit_account_id).to be_nil
+        expect(subject.recurrence_id).to eq(8833)
+        expect(subject.person_id).to be_nil
+        expect(subject.created_at).to eq(DateTime.parse("2015-10-20T16:27:15-02:00"))
+        expect(subject.updated_at).to eq(DateTime.parse("2015-10-20T16:27:15-02:00"))
```

Comment:
Do not use `DateTime.parse` without zone. Use one of `Time.zone.parse`, `DateTime.parse.current`, `DateTime.parse.in_time_zone`, `DateTime.parse.utc`, `DateTime.parse.getlocal`, `DateTime.parse.iso8601`, `DateTime.parse.jisx0301`, `DateTime.parse.rfc3339`, `DateTime.parse.to_i`, `DateTime.parse.to_f` instead.